### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Manynik**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,18 @@
...
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        return
+
+    source = parts[1]
+    target = parts[2]
```
Comment (viktoria-rybenchuk) [inline app/main.py:10]: You can use tuple unpacking to assign values from the `parts`

File: app/main.py [hunk 2]
```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        return
```
Comment (viktoria-rybenchuk) [inline app/main.py:7]: You can simplify the task with one `return`

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
Follow comments and read checklist

File: app/main.py [hunk 3]
```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    source, target = parts[1], parts[2]
```
Comment (viktoria-rybenchuk) [inline app/main.py]: First, check if the length of `parts` is 3 and you can unpack command as well

File: app/main.py [hunk 4]
```diff
@@ -1 +1,14 @@
...
+    if len(parts) != 3 or parts[0] != "mv" or source == target:
+        return
+
+    if os.path.dirname(target) != "":
```
Comment (viktoria-rybenchuk) [inline app/main.py:10]: You can simply check it with `if target`.

File: app/main.py (hunk 3 above)
Comment (viktoria-rybenchuk) [inline app/main.py]:
```suggestion
   command,  source, target = parts
```

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 4 above)
Comment (Manynik) [reply app/main.py:10]: i don't understand

File: app/main.py (hunk 3 above)
Comment (Manynik) [reply app/main.py]: how i can do it with one "return"?

File: app/main.py (hunk 4 above)
Comment (viktoria-rybenchuk) [reply app/main.py:10]: first you need  make this condition
Screenshot from 2023-09-22 14-41-55

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**